doc/changelog/02-specification-language/19029-master+sealed-attribute.rst

```diff
@@ -0,0 +1,8 @@
+
+- **Added:**
+  New attributes :attr:`sealed` and :attr:`defined` allow to change
+  the default opacity (now called sealing to prevent the confusion
+  with the notion of opacity implemented by the command :cmd:`Opaque`)
+  of a definition or theorem
```

Comment:
```suggestion
  Attributes :attr:`sealed` and :attr:`defined` to set
  the opacity of a definition or theorem.  :attr:`sealed` makes the object
  immutably opaque, unlike the :cmd:`Opaque` command, which makes
  an object mutably opaque
```

Some of the wording was not idiomatic
Also "the confusion with the notion of opacity" doesn't tell you what the difference in notions is, so vague